templates/layouts/beluga-install.hbs

```diff
@@ -66,34 +66,55 @@
-      <br>
-      <br>Enable ADB on your watch with the following steps:
-        <ol>
-          <img src="{{assets}}/img/install-beluga-1.png" class="install-centered-img"/>
-          <li>Open the settings app and go to the <b>System</b> and then <b>About</b> page.</li>
-          <li>Tap the <b>Build number</b> 7 times until Developer options become available.</li>
-
-          <img src="{{assets}}/img/install-beluga-2.png" class="install-centered-img"/>
-          <li>Go back to the settings menu and then to the <b>Developer options</b> page.</li>
-          <li>Enable <b>ADB debugging</b> and <b>OEM unlock</b>.</li>
-        </ol>
-      <br><b>Finally, to unlock the bootloader</b> enter the following commands in a terminal with your watch connected:</li>
+      <h2>1. Unlock your bootloader</h2>
+      Installing AsteroidOS requires an unlocked bootloader.
+      <br>Accessing the fastboot bootloader menu on your watch can be achieved via ADB in Wear OS, as described below in step 1.1.
+      <br>You can skip to step 1.2 if you are familiar with the manual method to access the bootloader menu. The methods are explained on the <a href="https://asteroidos.org/wiki/useful-commands/#boot-to-fastboot-bootloader-menu">useful commands page</a>.
+      <h3>1.1 Enable ADB on your watch with the following steps:</h3>
+      <div class="unlock-adb-wrapper">
+        <div class="unlock-step-wrapper">
+          <img src="{{assets}}/img/install-beluga-01.webp" class="unlock-step-img">
+          <div class="unlock-step-text">1. Open the <b>settings</b> app</div>
+        </div>
+        <div class="unlock-step-wrapper">
+          <img src="{{assets}}/img/install-beluga-02.webp" class="unlock-step-img">
+          <div class="unlock-step-text">2. Open the <b>System</b> page</div>
+        </div>
+        <div class="unlock-step-wrapper">
+          <img src="{{assets}}/img/install-beluga-03.webp" class="unlock-step-img">
+          <div class="unlock-step-text">3. Open the <b>About</b> page</div>
+        </div>
+        <div class="unlock-step-wrapper">
+          <img src="{{assets}}/img/install-beluga-04.webp" class="unlock-step-img">
+          <div class="unlock-step-text">4. Tap the <b>Build number</b> 7 times to reveal Developer options</div>
+        </div>
+        <div class="unlock-step-wrapper">
+          <img src="{{assets}}/img/install-beluga-01.webp" class="unlock-step-img">
+          <div class="unlock-step-text">5. Open the <b>Settings</b> app again</div>
+        </div>
+        <div class="unlock-step-wrapper">
+          <img src="{{assets}}/img/install-beluga-05.webp" class="unlock-step-img">
+          <div class="unlock-step-text">6. Open the <b>Developer options</b> page</div>
+        </div>
+        <div class="unlock-step-wrapper">
+          <img src="{{assets}}/img/install-beluga-06.webp" class="unlock-step-img">
+          <div class="unlock-step-text">7. Enable <b>ADB debugging</b> and <b>OEM unlock</b></div>
```

Comment:
Iiuc, this unlocks the bootloader right ?

Is step 1.2 necessary then ?

Reply:
The OEM unlock switch is needed to allow unlock. `fastboot oem unlock` stilll needs to be issued but would fail if the option is not set.

Reply:
Ah I see, thank you